tests/suite/common/paths.rs

```diff
@@ -0,0 +1,19 @@
+use std::path::Path;
+
+#[cfg(unix)]
+pub const MINIMAL_PATH: &str = "/usr/bin:/bin:/usr/sbin:/sbin";
```

Comment:
This is missing a Windows equivalent

Reply:
Yes, that's intentional and done elsewhere in this same fashion, for example in [Locations::new](https://github.com/dfinity/dfxvm/blob/main/src/locations.rs#L75).  This is in order to implement on Linux/OSX now, without having to research/resolve/test the Windows parts, but also to get a compile-time error rather than a runtime error on Windows.